src/einsteinpy/tests/test_metric/test_kerr.py

```diff
@@ -54,7 +54,7 @@
 def test_calculate_trajectory(
     pos_vec, vel_vec, time, M, a, start_lambda, end_lambda, OdeMethodKwargs
 ):
-    _scr = schwarzschild_radius(M).value
+    _scr = utils.schwarzschild_radius_dimensionless(M)
```

Comment:
This will now look like :
```suggestion
    _scr = schwarzschild_radius_dimensionless(M)
```